### PR TITLE
Fix Swagger Linting issues reported

### PR DESCRIPTION
**Pull request by riserrad**
We received the following request to fix Linting issues in our Swagger Resource Manager spec:

R4005:UniqueXmsEnumName
R4010:RequiredDefaultResponse

This change is to fix specifically those two.

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [x] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by riserrad** [comment]
Hi @njuCZ , we made this change to the swagger spec as we received a notification saying that we should do so. It seems that it produces a breaking change.

What do you suggest us doing in this case? Thanks

**Comment by riserrad** [comment]
> Hi @riserrad, one or multiple breaking change(s) is detected in your PR. Please check out the breaking change(s), and provide business justification in the PR comment and @ PR assignee why you must have these change(s), and how external customer impact can be mitigated. Please ensure to follow [breaking change policy](https://aka.ms/AzBreakingChangesPolicy) to request breaking change review and approval before proceeding swagger PR review.
> If you think it is false positive breaking change, please provide the reasons in the PR comment, report to Swagger Tooling Team via https://aka.ma/swaggerfeedback.

Hi @njuCZ , as described above, this change is to make this spec comply with the new rule R4005 that states that `x-ms-enum` should not have the same name. Both are `modelAsString: true`, and their property name haven't changed.

File: specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/stable/2020-05-15/timeseriesinsights.json [hunk 1]
```diff
@@ -57,6 +57,12 @@
             "schema": {
               "$ref": "#/definitions/OperationListResult"
             }
+          },
+          "default": {
+            "description": "HTTP 404 (Not Found): The list of operations could not be found.",
```
Comment (jorgecotillo) [inline specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/stable/2020-05-15/timeseriesinsights.json:62]: nit, but I usually see a generic error message here such as: "Error response describing why the operation failed." although I also noticed other services mentioning 404 as part of the description. Will check if there's any guidance around unifying these descriptions, but nothing to block this PR.

**Review by jorgecotillo — APPROVED** [review]
LGTM

File: specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/stable/2020-05-15/timeseriesinsights.json (hunk 1 above)
Comment (riserrad) [reply specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/stable/2020-05-15/timeseriesinsights.json:62]: Thanks, @jorgecotillo . I'd appreciate any guidance on this. If you find anything, could you mail me at my MSFT mail? 

Thanks!

**Review by njuCZ — APPROVED** [review]

**Pull request merged**